src/SharpBrick.PoweredUp/Protocol/LegoWirelessProtocol.cs

```diff
@@ -84,6 +87,11 @@ public async Task SendMessageAsync(LegoWirelessMessage message)
             throw;
         }
     }
+    
+    private bool UseCachedInformation()
```

Comment:
The only feedback here: Why using a method here instead of a property?

Reply:
Agreed, habit I guess :)